apps/src/gamelab/GameLabJr.interpreted.js

```diff
@@ -0,0 +1,315 @@
+/* This file is only executed within JSInterpreter */
```

Comment:
I _really_ like the convention of `.interpreted.js` to denote libraries that run inside the interpreter!